instaclustr/data_source_kafka_topic_list.go

```diff
@@ -0,0 +1,44 @@
+package instaclustr
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceKafkaTopicList() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceKafkaTopicListRead,
+
+		Schema: map[string]*schema.Schema{
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"topics": &schema.Schema{
```

Comment:
Just wondering - why is the topic list here limited to a list of strings? Does the API not return the other information we feed into the topic resource?

Reply:
No, for reading topic list, it only returns the list of topics, no other information.